scripts/convert-gitops-template.sh

```diff
@@ -33,6 +33,7 @@ convert() {
     replace "values\.modelEndpointSecretKey" ".Values.model.modelEndpointSecretKey" "$input_file"
     replace "values\.existingModelServer" ".Values.model.existingModelServer" "$input_file"
     replace "values\.dbRequired" ".Values.model.dbRequired" "$input_file"
+    replace "values\.maxModelLength" ".Values.model.maxModelLength" "$input_file"
```

Comment:
at some point can you confirm if you ran `shellcheck` against this file for the changes here?

Reply:
I think I'll add today one or two changes for the bearer auth case and then I'll also run shellcheck

Reply:
I ran the shellcheck and I added a minor suggestion. (see 
99d8f42) 
I'll resolve this one!